### ISSUE TITLE
Missing Tab order focus after Close Button

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/helix-sidekick-extension#251]
**Description**
Tab Focus is lost after moving from the close button. The user needs to press two missing tabs to reach the next focussable link "gdrive"

**To Reproduce**
1. Go to URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Use the sidekick extension
3. Use the tab to reach the close button
4. After reaching the close button, press two empty tabs to reach link "gdrive"

**Expected behavior**
The focussable elements should be in the tab order.

**Screenshots**

**Additional context**
https://www.w3.org/TR/WCAG21/#focus-order

**Why it matters?**
When users navigate through a web page, they expect to encounter controls and other content in an order that makes sense and makes it easy to use the page's functionality. Lost or Poor focus order can be disorienting to people who use screen readers or screen magnifiers and to people with reading disorders. Lost or Poor focus order can also make it difficult or even painful for people who use keyboards because of mobility impairments.

**Comment by rofe** [comment]
@puja-iaccessible the tab index follows the natural DOM order. After the close button, you leave the scope of the sidekick and continue on to the first focusable element on the underlying web page. By default, this link may be hidden underneath the sidekick (see also #252).

**Comment by rofe** [comment]
Closing this as duplicate of #252.
